examples/demo_actor/README.md

```diff
@@ -1,63 +1,11 @@
-# Dapr SDK
+# DemoActor
 
-> WIP - Porting from [dotnet-sdk](https://github.com/dapr/dotnet-sdk) to python
-> 
-> Dapr core team does not have the official plan to support python-sdk now, except for the auto-generated gRPC client. but we're always welcoming any contribution.
-
-## Structures of Python SDK
-
-* [dapr/actor](../dapr/actor): Actor Framework
-* [dapr/clients](../dapr/clients): HTTP clients for Dapr building blocks (maybe we need to merge gRPC proto client to this directory)
-* [dapr/serializers](../dapr/serializers): serializer/deserializer
-* [dapr/conf](../dapr/conf): Configuration
-* [flask_dapr](../flask_dapr): flask extension for Dapr
-* [tests](../tests/): unit-tests
-* [examples/demo_actor](../examples/demo_actor): demo actor example
-
-## Status
-
-* [x] Initial implementation of Actor Runtime/Manager/Proxy
-* [x] Actor service invocation
-* [x] RPC style actor proxy
-* [x] Flask integration for Dapr Actor Service
-* [x] Example for Actor service invocation
-* [ ] Complete tox.ini setup
-* [ ] Actor state management
-* [ ] Actor timer
-* [ ] Actor reminder
-* [ ] Package Dapr Actor SDK
-* [ ] Create gRPC and HTTP rest clients for Dapr
-* [ ] Flask extensions for Dapr State/Pubsub/Bindings
-* [ ] Package Dapr SDK
-
-## Developing
-
-### Prerequisites
+## Prerequisites
```

Comment:
Need a few sentences here on what this example is and what it does.

Reply:
This is the PR merging to feature branch. when I merge to master, I will add more comments on that. I am gonna add more examples to demo_actor.